Header header
geometry_msgs/Point32[] keypoints
geometry_msgs/Point32 offset
sensor_msgs/CompressedImage image